From 2b2e0b46a5d84600805515772f4ce0310d74d81c Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 12 May 2021 10:31:27 +0300
Subject: [PATCH] [SW-42169] gaudi: pll: adjusting pci_pll div2 value

Lowering the clock produced by pci pll div2 output from 166Mhz to
103.75Mhz due to unstable pci phy operation.
The unstable operation is obsereved when the host pcie insteface
goes through multiple enable/disable cycles, at some point the
pcie phy in gaudi device goes to power down mode unexpectedly.

Change-Id: I661cbc069203fa52b6e18b826ed2dc3d4b048e2e
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/gaudi/pll.c b/gaudi/pll.c
index 7522f73..07520f7 100644
--- a/gaudi/pll.c
+++ b/gaudi/pll.c
@@ -70,7 +70,7 @@
 	}, {
 		.name = "PCI", .base = mmPSOC_PCI_PLL_BASE & ~HL_CFG_BASE,
 		.nr = 4, .nf = 331, .od = 3, .nb = 165, /* 830 */
-		.div_factor = { 0, 0, 4, 9 },
+		.div_factor = { 0, 0, 7, 9 },
 		.div_sel = { HL_PLL_PLL_CLK,
 			     HL_PLL_PLL_CLK,
 			     HL_PLL_DIV_PLL,
